src/app/pages/token-detail/token-detail.component.html

```diff
@@ -0,0 +1,5 @@
+<app-color-card-item
```

Comment:
@AndreasGassmann we should display the same content as in the `color-history-modal`, probably the best way is to created a component to display on the detail page or in the modal?